csrc/evaluator_common.cpp

```diff
@@ -353,6 +353,14 @@ NaiveValueMachine::NaiveValueMachine(PrecomputedValues& precomputed_values)
         makeUnaryOp(uop);
       } else if (auto bop = dynamic_cast<BinaryOp*>(def)) {
         makeBinaryOp(bop);
+      } else if (auto top = dynamic_cast<TernaryOp*>(def)) {
+        makeTernaryOp(top);
+      } else if (auto lsop = dynamic_cast<LoadStoreOp*>(def)) {
+        NVF_ERROR(
+            lsop->opType() == LoadStoreOpType::Set,
+            "NaiveValueMachine: unsupported LoadStoreOpType: ",
+            lsop->opType());
+        makeSetOp(lsop);
```

Comment:
Unused in this PR currently but we already had `SET_OP`, so this lets along with `makeSetOp` us evaluate `set`.